pkg/components/util/install.go

```diff
@@ -140,3 +135,28 @@ func InstallAsRelease(name string, c components.Component, kubeconfig string) er
 
 	return nil
 }
+
+// HelmActionConfig creates initialized Helm action configuration.
+func HelmActionConfig(ns string, kubeconfig string) (*action.Configuration, error) {
+	actionConfig := &action.Configuration{}
+
+	// TODO: Add some logging implementation? We currently just pass an empty function for logging.
+	if err := actionConfig.Init(kube.GetConfig(kubeconfig, "", ns), ns, "secret", func(format string, v ...interface{}) {}); err != nil {
+		return nil, fmt.Errorf("failed initializing helm: %v", err)
```

Comment:
Should we use `%w` here?
```suggestion
		return nil, fmt.Errorf("failed initializing helm: %w", err)
```

Reply:
Yes, done.